packages/components/src/autocomplete/src/Autocomplete.tsx

```diff
@@ -419,6 +424,7 @@ export function InnerAutocomplete(props: InnerAutocompleteProps) {
                         readOnly,
                         ref: triggerRef,
                         role: "combobox",
+                        size,
```

Comment:
A default value should not be added, for example `md`.

Reply:
I'm confused, are you guys saying that size should or should NOT have a default value?
Right now, I see that it does not have one, but usually orbiter has "md" as a default. Like Francis just said.

Reply:
We don't have to set one as md is the default

Reply:
where does it do that, only in the css? I feel like we should still make it obvious here.